[Bug]: ZeroClaw compilation fails on DigitalOcean — Rust 2021 edition string literal errors

### What happened?

Running `spawn zeroclaw digitalocean` fails during ZeroClaw compilation (`v0.1.6`) with multiple Rust compiler errors in `src/security/leak_detector.rs`:

```
error: prefix `key` is unknown
  --> src/security/leak_detector.rs:92:99
   |
92 |                 (Regex::new(r"api[_-]?key[=:]\s*['\"]*[a-zA-Z0-9_-]{20,}").unwrap(), "Generic API key"),
   |                                                                                                   ^^^ unknown prefix

error: prefix `ID` is unknown
   --> src/security/leak_detector.rs:109:76
    |
109 |                 (Regex::new(r"AKIA[A-Z0-9]{16}").unwrap(), "AWS Access Key ID"),
    |                                                                            ^^ unknown prefix

error: unknown start of token: \
   --> src/security/leak_detector.rs:110:68

error[E0762]: unterminated character literal
   --> src/security/leak_detector.rs:110:72
```

**Root cause:** The string literals in `leak_detector.rs` use prefixed identifiers (e.g., `key"`, `ID"`) that are reserved in **Rust 2021 edition**. The raw string `r"..."` regex patterns contain unescaped quotes that confuse the Rust lexer.

**Reproduction:**
```bash
spawn zeroclaw digitalocean --name "test-zeroclaw"
```

This is an upstream ZeroClaw source issue — the regex patterns in `leak_detector.rs` need to use raw string literals with delimiters (e.g., `r#"..."#`) to avoid conflicts with the Rust 2021 reserved prefix rules.

---

_Filed from Slack by SPA_

--- a/cli/package.json
+++ b/cli/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@openrouter/spawn",
-  "version": "0.8.4",
+  "version": "0.8.5",
   "type": "module",
   "bin": {
     "spawn": "cli.js"

--- a/cli/src/shared/agent-setup.ts
+++ b/cli/src/shared/agent-setup.ts
@@ -398,7 +398,7 @@ export function openCodeInstallCmd(): string {
 // ─── Default Agent Definitions ───────────────────────────────────────────────
 
 const ZEROCLAW_INSTALL_URL =
-  "https://raw.githubusercontent.com/zeroclaw-labs/zeroclaw/a117be64fdaa31779204beadf2942c8aef57d0e5/scripts/install.sh";
+  "https://raw.githubusercontent.com/zeroclaw-labs/zeroclaw/a117be64fdaa31779204beadf2942c8aef57d0e5/scripts/bootstrap.sh";
 
 export function createAgents(runner: CloudRunner): Record<string, AgentConfig> {
   return {
@@ -486,7 +486,7 @@ export function createAgents(runner: CloudRunner): Record<string, AgentConfig> {
         installAgent(
           runner,
           "ZeroClaw",
-          `curl -LsSf ${ZEROCLAW_INSTALL_URL} | bash -s -- --install-rust --install-system-deps`,
+          `curl -LsSf ${ZEROCLAW_INSTALL_URL} | bash -s -- --install-rust --install-system-deps --prefer-prebuilt`,
         ),
       envVars: (apiKey) => [
         `OPENROUTER_API_KEY=${apiKey}`,

--- a/manifest.json
+++ b/manifest.json
@@ -95,7 +95,7 @@
       "name": "ZeroClaw",
       "description": "Fast, small, fully autonomous AI assistant infrastructure \u2014 deploy anywhere, swap anything",
       "url": "https://github.com/zeroclaw-labs/zeroclaw",
-      "install": "curl -LsSf https://raw.githubusercontent.com/zeroclaw-labs/zeroclaw/a117be64fdaa31779204beadf2942c8aef57d0e5/scripts/install.sh | bash -s -- --install-rust --install-system-deps",
+      "install": "curl -LsSf https://raw.githubusercontent.com/zeroclaw-labs/zeroclaw/a117be64fdaa31779204beadf2942c8aef57d0e5/scripts/bootstrap.sh | bash -s -- --install-rust --install-system-deps --prefer-prebuilt",
       "launch": "zeroclaw agent",
       "env": {
         "OPENROUTER_API_KEY": "${OPENROUTER_API_KEY}",

